rules/Proj0800.md

```diff
@@ -7,41 +7,26 @@ you can leverage Central Package Management (CPM) features to do all of
 this from the ease of a single location.
 
 Hence, this rules advises to enable Central Package Management by
-enabling it with `<ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>`.
-It is advised to do this in the `Directory.Packages.props` file.
+enabling it with in the `Directory.Packages.props` file:
 
-## Non-compliant
-``` XML
-<Project Sdk="Microsoft.NET.Sdk">
- 
- <PropertyGroup>
-   <TargetFramework>net8.0</TargetFramework>
-  </PropertyGroup>
- 
-</Project>
-```
-
-## Compliant
 ``` XML
 <Project Sdk="Microsoft.NET.Sdk">
 
  <PropertyGroup>
-   <TargetFramework>net8.0</TargetFramework>
    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
  </PropertyGroup>
  
 </Project>
 ```
 
 When a solution has only a tiny number of project files, and/or hardly any
-package is shared amongst these projects, CPM might be not worth the
-effort. In that case it is also fine to explicitly disable it.
+packages shared amongst these projects, CPM might be not worth the
+effort. In that case it is also fine to explicitly disable it:
 
 ``` XML
 <Project Sdk="Microsoft.NET.Sdk">
 
  <PropertyGroup>
-   <TargetFramework>net8.0</TargetFramework>
```

Comment:
TargetFramework is valid in a project file. We might want to keep it here, to make it more clear that it is about the project file.